src/tools/Misc.cpp

```diff
@@ -33,3 +34,17 @@ string misc::getPartNameFromReference(gsoap_eml2_2::eml22__DataObjectReference *
 	return reference->ContentType.substr(reference->ContentType.rfind('=') + 1) + "_" + reference->Uuid + ".xml";
 }
 
+gsoap_resqml2_0_1::eml20__DataObjectReference* misc::eml22ToEml20Reference(gsoap_eml2_2::eml22__DataObjectReference* reference, soap* soapContext)
+{
+	gsoap_resqml2_0_1::eml20__DataObjectReference* result = gsoap_resqml2_0_1::soap_new_eml20__DataObjectReference(soapContext);
+	result->UUID = reference->Uuid;
+	result->Title = reference->Title;
+	result->ContentType = reference->ContentType;
+	if (reference->ObjectVersion != nullptr) {
```

Comment:
could also check if it is not empty neither.
An emtpy version does not mean anything and, imo, is like a null version.

Reply:
Done.